doc/source/dev/contributor/adding_notebooks.rst

```diff
@@ -0,0 +1,146 @@
+- **Internal links:** For internal cross-references such as links to SciPy
+  classes or functions, as well as intersphinx links, you can use the following
+  syntax: ``{role}`link text <reference>```, where ``role`` can be ``ref``,
+  ``class``, ``func`` or any other role you would use with reST. For example, to
+  link to the ``scipy.stats.bartlett`` function, use
+  ``{func}`scipy.stats.bartlett```.
+- **Formatting:** MyST Markdown supports standard Markdown formatting, such as
+  bold text, italic text, and code blocks. For example, to make text bold, you
+  can use double asterisks: ``**bold text**``. To make text monospace/formatted
+  as code, you can use *single* backticks: ```code```.
+- **Images:** If your notebook contains static images, you can include them in
+  the MyST Markdown file by using the following syntax:
+  ``![alt text](path/to/image.png)``. Images are usually stored in the same
+  folder as the MyST Markdown file, but you can also use relative paths to
+  reference images in other folders. Note that outputs of executed cells should
+  not be included in version control, as they will be automatically generated
+  when the notebook is executed during the SciPy documentation build.
+- **Linking to the MyST markdown pages:** MyST markdown supports adding link
+  anchors, which can be used to link to specific pages or document sections from
+  other documents. To add an anchor to a page, add ``(anchor_name)=`` to the
+  desired place in the document to be referenced. From other markdown pages, you
+  can link to this anchor using the following syntax: ``{ref}`anchor_name```.
+  From other reST pages, you can link to this anchor using the following syntax:
+  ``:ref:`anchor_name```.
+
+Making the notebook interactive
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to enable interactivity for your notebook when it is rendered in
+the SciPy documentation website, you need to add the following snippet at the
+beginning of the markdown file:
+
+.. code-block:: markdown
+
+    ```{eval-rst}
+    .. notebooklite:: <filename>.md
+       :new_tab: True
+    ```
+
+where ``<filename>.md`` should be replaced with the name of the file you are
+working with. This will create a button that allows users to open the notebook
+in a new tab and interact with it. See
+`Bartlett's test for equal variances <https://scipy.github.io/devdocs/tutorial/stats/hypothesis_bartlett.html>`__
+(and `its source <https://github.com/scipy/scipy/blob/main/doc/source/tutorial/stats/hypothesis_bartlett.md?plain=1>`__)
+for an example.
+
+.. note::
+
+    For simplicity, we don't want certain notebook cells to be visible in the
+    rendered documentation. We can hide them by adding ``"jupyterlite_sphinx_strip"``
+    to the tags section of the corresponding cell metadata. To do that in the
+    markdown file, add the following snippet right before cell you want to hide:
+
+    .. code-block:: markdown
+
+        +++ {"tags": ["jupyterlite_sphinx_strip"]}
```

Comment:
As the Markdown cells are not fenced the same way as other cells, we should mention here that the `+++` notation should come in pairs (otherwise, the entire content up to the following code cell would get hidden).